spec/unit/configurer_spec.rb

```diff
@@ -11,8 +13,8 @@
     Puppet[:report] = true
   end
 
-  it "should include the Fact Handler module" do
-    expect(Puppet::Configurer.ancestors).to be_include(Puppet::Configurer::FactHandler)
```

Comment:
This is a test code smell, as we're testing the structure of the code, not it's behavior. Better to test that when the run method is called, it collects facts, etc which is done else where in the test. Puppet unfortunately has lots of these tests, and I try to delete them as I run across them.

Reply:
Is this implicitly tested via other tests then?

Reply:
Yep, the test `should pass the url encoded facts and facts format as arguments when retrieving the catalog` verifies that the current set of facts are submitted with the catalog request. Ideally that test would be using `stub_request` instead of setting an expectation on the Indirection#find method, but I left that for another day.